swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/autoscalinggroups/AmazonAutoScalingGroup.kt

```diff
@@ -79,14 +79,6 @@ data class AmazonAutoScalingGroup(
     return getAddToLoadBalancerProcess()?.suspensionReason
   }
 
-  fun isDisabled(): Boolean {
```

Comment:
@aravindmd this creates a circular dependency.

Reply:
Can you explain this bit more ?

Reply:
`IS_DISABLED` is not a field on the amazon autoscaling group. It's something that the handler was responsible to set.

Reply:
Thanks , that makes sense now.